support/Pipelines/Bbh/InitialData.yaml

```diff
@@ -133,6 +133,7 @@ Amr:
   Verbosity: Verbose
   Criteria: []
   Policies:
+    EnforceTwoToOneBalanceInNormalDirection: true
```

Comment:
Maybe set all these options to `false`? To me at least that's the expected behavior of AMR. Do we even have a use case for `true`?

Reply:
I set them to `true` to reproduce previous behavior.  Until setting them to `false` is more extensively tested, I'd rather keep them `true` for now.

Reply:
Ok Im just worried that this will remain `true` and propagate to many input files and people will get unexpected behavior when trying AMR

Reply:
I think in evolution problems we probably want it true. Geoffrey had issues if the grid size changed too much form one element to the next in the normal direction to the interface.

Reply:
That's actually what I'm trying to prevent as well, in a domain that has differently sized blocks. For example, think of the thermal noise problem with thin layers on top of a substrate. You want to h-refine the substrate multiple times in z-direction, but each layer stays at h-refinement level 0.

Anyway, we can merge the PR like this and just have to keep in mind to experiment with this option.